tests/test_atomic_requests.py

```diff
@@ -124,22 +138,8 @@ def test_api_exception_rollback_transaction(self):
     connection.features.uses_savepoints,
     "'atomic' requires transactions and savepoints."
 )
+@override_settings(ROOT_URLCONF='tests.test_atomic_requests')
```

Comment:
btw, this test did not fail despite the urls property having been fully deprecated in 1.10